README.md

```diff
@@ -19,8 +19,8 @@ exceptions/differences/extensions (:white_check_mark: are the implemented sniffs
 - Keep the nesting of control structures per method as small as possible
 - Prefer early exit over nesting conditions or using else
 - Abstract exception class names and exception interface names should be suffixed with `Exception`
-- :white_check_mark: Abstract classes should not be prefixed with `Abstract`
-- :white_check_mark: Interfaces should not be suffixed with `Interface`
+- :white_check_mark: Abstract classes should not be suffixed with `Abstract`
```

Comment:
In fact, I'm not sure… do we, or do we not want to forbid the `Abstract` prefix. My PR was originally just about `Exception`, see with @carusogabriel why these other rules were added.

Reply:
The sniff reports both so this chane is not enough

Reply:
@kukulich I don't know if we want to forbid the prefix or not.

Reply:
It is currently forbidden in 4.0.0...

Reply:
Oh well…